### ISSUE TITLE
Téléquebec problem and not updating

### ISSUE THREAD

**Issue by mixeta** [SiCKRAGETV/sickrage-issues#576]
Hello
Sorry If I don't use the right way of writing an issue but I have a problem with sickrage and I cannot find the solution... I'm new with everything of code but at least I was able to have sickrage working for a while. Now I have this problem on my log:

http://pastebin.com/yXv5jdYt

SR Version:	BRANCH: (develop) / COMMIT: (401cb666016e45b42fe675bafdf7908ad6c2b9bb) 
Python Version:	2.7.3 (default, Jan 4 2013, 13:44:41) [GCC 4.6.3]

I have a WD MY CLOUD 3TB.

Any idea???? I tried several things but I can't find the solution...
​Thanks!

**Comment by abeloin** [comment]
Make sure you use utf8 in your locale, see https://sickrage.tv/forums/forum/help-support/general-support/12825-ensuring-your-locale-s-are-set-properly-to-avoid-unicode-issues

**Comment by mixeta** [comment]
Thank you for your answer abeloin.

I'll check if I find the files easily... Anyone one where to find this on WD MY CLOUD? Just to be sure... 

And WD MY CLOUD has a unix operating system?

Again, sorry for the stupid questions but I'm just a newbie with code...

**Comment by abeloin** [comment]
Ok I didn't realized it was a NAS. Yes it use linux.

You have 2 options: change your locale(if supported) or wait until SiCKRAGETV/SickRage#1224 is merge in develop.

Want to change your locale and you have access via ssh to your device? If so, type to following command: ```locale```.
It should list your current locale.
To see the available locale type to following command: ```locale -a```.
It should list all the locale installed but since it's a nas they may have decided not to include UTF.
Then follow https://sickrage.tv/forums/forum/help-support/general-support/12825-ensuring-your-locale-s-are-set-properly-to-avoid-unicode-issues

**Comment by mixeta** [comment]
Hello Again!
Thanks for all the info!

I tried what you said about the locale and this is the log:

locale: http://pastebin.com/0HafHYgt

locale -a: http://pastebin.com/KqmpeBVK

Everything look okay and I have UTF...

And then, I try to do the things on the link you sent me and I cannot find:
"Add the following lines of code to your ~/.bashrc and ~/.profile files".

I don't know where are these files...

If you could help me one more time I will appreciate a lot.
Thanks abeloin!

2015-02-10 17:46 GMT+01:00 Alexandre Beloin <notifications@github.com>:

> Ok I didn't realized it was a NAS. Yes it use linux.
>
> You have 2 options: change your locale(if supported) or wait until
> SiCKRAGETV/SickRage#1224
> <https://github.com/SiCKRAGETV/SickRage/pull/1224> is merge in develop.
>
> Want to change your locale and you have access via ssh to your device? If
> so, type to following command: locale.
> It should list your current locale.
> To see the available locale type to following command: locale -a.
> It should list all the locale installed but since it's a nas they may have
> decided not to include UTF.
> Then follow
> https://sickrage.tv/forums/forum/help-support/general-support/12825-ensuring-your-locale-s-are-set-properly-to-avoid-unicode-issues
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/SiCKRAGETV/sickrage-issues/issues/576#issuecomment-73734750>
> .
>



-- 
Romà Bertran Navés
+ 33 6 43 96 21 17

**Comment by abeloin** [comment]
If they don't exist create them using vi: ```vi ~/.bashrc```
Press i to be able to insert text.
When press ESC then :wq

**Comment by mixeta** [comment]
Hello Alexandre
Thanks for everything.
I tried what you told me but still it doesn't work... I tried to insert the
text but it doesn't do anything. Maybe I insterded on the wrong line but I
don't know how to delete things on bashrc and profile so I think it's
better to not touch anymore and  wait until they merge everything. I'll
continue using the master version.

Thank you again for helping me!

2015-02-10 19:25 GMT+01:00 Alexandre Beloin <notifications@github.com>:

> If they don't exist create them using vi: vi ~/.bashrc
> Press i to be able to insert text.
> When press ESC then :wq
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/SiCKRAGETV/sickrage-issues/issues/576#issuecomment-73753746>
> .
>



-- 
Romà Bertran Navés
+ 33 6 43 96 21 17
